libs/components/lists/src/lib/modules/filter/filter-button.component.spec.ts

```diff
@@ -22,15 +22,15 @@ describe('Filter button', () => {
     fixture.detectChanges();
   });
 
-  function getButtonEl() {
+  function getButtonEl(): HTMLButtonElement {
```

Comment:
is it possible for this to ever be undefined?

Reply:
I don't think so? The fixture has a FilterButtonComponent child, which has a button child so searching for that class should always return a result.